network/load-balancer/concepts.mdx

```diff
@@ -169,6 +169,16 @@ Read more about HTTP/2 and HTTP/3 and how they can be configured on Scaleway Loa
 
 <Concept>
 
+## Kubernetes Load Balancer
+
+When you create a `LoadBalancer` service within your [Scaleway Kubernetes cluster](/containers/kubernetes/quickstart/), the Scaleway Cloud Controller Manager will spin up an external Scaleway Load Balancer with the correct configuration to expose the specified service in your cluster.
+
+The process for creating and managing Load Balancers for Kubernetes clusters is very specific - you **cannot** create or edit these via the Scaleway console or API. For full instructions on how to correctly create and configure a Scaleway Load Balancer for a Kubernetes cluster, see our [dedicated documentation](/containers/reference-content/kubernetes-load-balancer/).
```

Comment:
@RoRoJ the link to the Kubernetes doc doesn't work 

> [dedicated documentation](/containers/reference-content/kubernetes-load-balancer/)

I don't know if it's because it's the staging environment or because the documents have been reorganized and the link target doesn't exist anymore.

Reply:
Fixed